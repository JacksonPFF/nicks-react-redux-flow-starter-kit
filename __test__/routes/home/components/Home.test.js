import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../../../../src/routes/home/components/Home.component';

describe('Home Component', () => {
  const component = renderer.create(
    <Home />
  );
  it('Component renders', () => {
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
