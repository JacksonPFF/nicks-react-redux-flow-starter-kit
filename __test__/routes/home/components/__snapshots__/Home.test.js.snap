// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component Component renders 1`] = `
<div
  className="home"
>
  <img
    alt="React/Redux Serverless"
    onClick={[Function]}
    src="/images/koala.png"
  />
  <div
    className="ant-row"
    style={undefined}
  >
    <div
      className="ant-col-xs-8 ant-col-md-8"
    >
      <button
        className="ant-btn ant-btn-primary ant-btn-background-ghost"
        onClick={[Function]}
        onMouseUp={[Function]}
        type="button"
      >
        <span>
          Click
        </span>
      </button>
    </div>
    <div
      className="ant-col-xs-8 ant-col-md-8"
    >
      <button
        className="ant-btn ant-btn-danger ant-btn-background-ghost"
        onClick={[Function]}
        onMouseUp={[Function]}
        style={
          Object {
            "height": "100% ",
          }
        }
        type="button"
      >
        <span>
          Double Click 
        </span>
        <br />
        <small>
          (1s delay)
        </small>
      </button>
    </div>
    <div
      className="ant-col-xs-8 ant-col-md-8"
    >
      <span
        className="ant-badge"
        title={null}
      >
        Counter
      </span>
    </div>
  </div>
</div>
`;

exports[`Home Component Renders 1`] = `
<div
  className="home"
>
  <img
    alt="React/Redux Serverless"
    onClick={[Function]}
    src="/images/koala.png"
  />
  <div
    className="ant-row"
    style={undefined}
  >
    <div
      className="ant-col-xs-8 ant-col-md-8"
    >
      <button
        className="ant-btn ant-btn-primary ant-btn-background-ghost"
        onClick={[Function]}
        onMouseUp={[Function]}
        type="button"
      >
        <span>
          Click
        </span>
      </button>
    </div>
    <div
      className="ant-col-xs-8 ant-col-md-8"
    >
      <button
        className="ant-btn ant-btn-danger ant-btn-background-ghost"
        onClick={[Function]}
        onMouseUp={[Function]}
        style={
          Object {
            "height": "100% ",
          }
        }
        type="button"
      >
        <span>
          Double Click 
        </span>
        <br />
        <small>
          (1s delay)
        </small>
      </button>
    </div>
    <div
      className="ant-col-xs-8 ant-col-md-8"
    >
      <span
        className="ant-badge"
        title={null}
      >
        Counter
      </span>
    </div>
  </div>
</div>
`;
